starknet_py/tests/e2e/block_test.py

```diff
@@ -58,11 +60,14 @@ async def test_block_with_tx_hashes_latest(
     assert isinstance(blk.transactions, list)
     assert map_contract_declare_hash in blk.transactions
     assert blk.block_hash is not None
-    assert blk.parent_block_hash is not None
+    assert blk.parent_hash is not None
     assert blk.block_number is not None
-    assert blk.root is not None
+    assert blk.new_root is not None
     assert blk.timestamp is not None
     assert blk.sequencer_address is not None
+    assert blk.l1_gas_price is not None
+    assert blk.l1_data_gas_price is not None
+    assert blk.l1_da_mode is not None
```

Comment:
Why check that these are not `None`? 

I assume as they're required that shouldn't ever happen, so maybe we should check the values instead, similarly to how it's done in `estimate_fee` tests below? No need to use specific values, `!= 0` should suffice for latest/pending blocks.

Reply:
I would consider these tests to be kind of a sanity check, given that we have limited capabilities to test the exact values. I've added slightly more specific asserts for `l1_gas_price`, `l1_data_gas_price` and `l1_da_mode`.